python/Converters/PycArrayComCC.h

```diff
@@ -246,7 +248,7 @@
   void ArrayCopy<String>::toPy (void* to, const String* from, size_t nr)
   {
     PyObject** dst = static_cast<PyObject**>(to);
-    for (size_t i=0; i<nr; i++) {
+    for (uInt i=0; i<nr; i++) {
```

Comment:
I'm not sure what type `uInt` is, but if it's `unsigned int` then this change will break conversions where `nr` is more than 2**32 on 64-bit systems.

Reply:
again not sure what happened there, it has been a long time ago.

Reply:
yes, this actually made the python-casacore tests fail. Fixed it! thanks Bruce!